gremlin-client/Cargo.toml

```diff
@@ -69,8 +69,10 @@ tokio = { version = "1", optional=true, features = ["full"] }
 features = ["serde", "v4"]
 version = "1.1.2"
 
-
-
+[dev-dependencies]
+rstest = "0.23.0"
+rstest_reuse = "0.7.0"
+serial_test = "3.1.1"
```

Comment:
Each test generally used labels to prevent tests from stepping on each other. However the parameterized serializers will run concurrently. So I used `serial_test` on tests that parameters that operate on the same test don't step on each other and cause flakey failures.